DRODLib/GameConstants.h

```diff
@@ -27,12 +27,18 @@
 #ifndef GAMECONSTANTS_H
 #define GAMECONSTANTS_H
 
+#include <map>
+#include <unordered_map>
```

Comment:
As a rule of thumb, put these std include files below all the other includes, so that the files included lower down don't inadvertently satisfy a dependency they're not including themselves (but should be), by having these expanded into the source file above them. If that makes sense.